apollo-api/src/main/java/com/apollostack/api/GraphQLQuery.java

```diff
@@ -0,0 +1,10 @@
+package com.apollostack.api;
+
+import java.util.List;
+
+public interface GraphQLQuery<T> {
```

Comment:
It doesn't make sense to add a generic type to this interface if none of its methods are bound to `<T>`. Looks like we can remove it for now